Source/WarpX.cpp

```diff
@@ -1572,7 +1573,7 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
         full_rho_fp[lev] = std::make_unique<MultiFab>(amrex::convert(ba_full,rho_nodal_flag),dm_full,2*ncomps,ngRho,tag("full_rho_fp"));
     }
 
-    if (do_electrostatic == ElectrostaticSolverAlgo::LabFrame)
+    if (!external_solver && do_electrostatic == ElectrostaticSolverAlgo::LabFrame)
```

Comment:
Same comment as above. The `if external_solver` should only wrap the parts that initialize `full_phi_fp`.